### DISCUSSION TITLE
List of notes with wrong answers

### DISCUSSION THREAD

**Discussion by vrmneto** [st3v3nmw/obsidian-spaced-repetition#859] [General]
I'm using the "Spaced Repetition" plugin and I'm looking for some way to be able to list the notes on the flashcards that I made a mistake in reviewing.

when a note is designated as "hard" the repetition parameters return to very low values, such as 1, day. In this sense, I imagined that it would be possible to use these parameters and use them to define the error
A code is even used, I think it's HTML (I'm not in the computing field, so I can't identify it).
this is an example from my vault ""

Is it possible?

**Comment by ronzulu** [comment]
Fortunately Obsidian's search mechanism is quite versatile and supports "regular expressions". It's a very powerful, compact and precise notation for specifying what strings should be considered a match. It also can be difficult for humans to interpret what is being specified.

So the following specifies the HTML comment used by the plug-in to store flashcard review data. Specifically, those that have an interval of 1 day:
 `/<!--SR:.+,1,.*-->/`

With results such as:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/62048709-ac01-4f73-8ba8-4c20b211a17e)

Note that if you use the "Save scheduling comment on the same line as the flashcard's last line" option, Obsidian displays some context which you might find useful, e.g.:
![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/77467ecd-f7fd-4021-9dd7-11a1524435fe)

Because of the flexibility of regular expressions, you can specify a range of interval values, e.g. 1 through 3 days would be:
`/<!--SR:.+,[1-3],.*-->/`

(Note this regular expression would only work with single digit intervals)

Cheers
Ronny

**Comment by vrmneto** [comment]
Man, thank you. 

You really helped me here. 
I was struggling with that all the week.
I wil bother you with just one more thing. Can we create a new note with dataview showing the links of the notes that are in the pentameters "//"you wrote in HTML?

Thank you again. Really

**Comment by ronzulu** [comment]
Glad that this helped.

> when a note is designated as "hard" the repetition parameters return to very low values, such as 1, day. In this sense, I imagined that it would be possible to use these parameters and use them to define the error A code is even used, I think it's HTML (I'm not in the computing field, so I can't identify it).

Actually, what you wrote here gave me the idea to use the Obsidian search. Otherwise I'd only think of a solution that involved code modifications to the plug-in, which wouldn't happen any time soon.

There have been some suggested enhancements such as [[FEAT] Flashcards Review Schedule (Queue) Sidebar](https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/746) . If you would find that useful, you might want to add a comment on that issue.

> I wil bother you with just one more thing. Can we create a new note with dataview showing the links of the notes that are in the pentameters "//"you wrote in HTML?

Sorry, I've used dataview, but not an expert. You might want to ask on the support forum for that plug-in. Also, you might want to have a peek at the list of community plug-ins. There might be something there there that could help.

Cheers
Ronny
